### PR TITLE
Add 5.20.1 release notes

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds release notes for 5.20.1

## Motivation and Context
Support 5.20.1 patch release

### CODE REVIEW

**Comment by palourde** [comment]
@hillaryfraley I was remembered by Cyril that we also fixed the display of percentage for memory & cpu usage!

Additionally, processes names are no longer capitalized too.

**Comment by palourde** [comment]
@hillaryfraley Do you want to add a note about the % fix?

**Comment by hillaryfraley** [comment]
@palourde yes I'm sorry, I was in the middle of adding it when the meeting started.
Edit: ok I added it

File: content/sensu-go/5.20/release-notes.md [hunk 1]
```diff
@@ -60,6 +61,19 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+**May 15, 2020** &mdash; The latest release of Sensu Go, version 5.20.1, is now available for download.
+
+This patch release includes a bug fix that affects the web UI federated homepage gauges when using the PostgreSQL datastore and several fixes for the data displayed in the web UI entity details.
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.20.1.
+
+**FIXES:**
+
+- ([Commercial feature][141]) Fixes a bug that prevented the federated homepage in the [web UI][153] from retrieving the keepalive and event gauges when PostgreSQL was configured as the event datastore.
```
Comment (hillaryfraley) [inline content/sensu-go/5.20/release-notes.md:74]:
```suggestion
- ([Commercial feature][141]) Fixes a bug that prevented the federated homepage in the [web UI][153] from retrieving the keepalive and event gauges when PostgreSQL was configured as the event datastore.
- ([Commercial feature][141]) The [memory_percent and cpu_percent processes attributes][143] are now properly displayed in the [web UI][153].
```

**Review by palourde — APPROVED** [review]
Perfect, thanks!

**Pull request merged**